packages/core/src/prompts/prompt.ts

```diff
@@ -263,7 +263,7 @@ export default class Prompt<TValue> {
 		this.output.write(cursor.move(-999, lines * -1));
 	}
 
-	private render() {
+	protected render() {
```

Comment:
We need to be able to call this manually, as we're not just rendering on keypress